packages/website/src/docs/pill.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Pill
+component: Pill
```

Comment:
Should this component be called `PillButton`? I feel it doesn't communicate well that it's a button.

Reply:
Yeah good point, another breaking change 🔥